presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.234
+=============
+
+.. warning::
+
+   There is a regression in the scheduler that has not been fixed in this release. It has been temporarily mitigated by setting
+   the default value of the configuration property ``use-legacy-scheduler`` to ``true``. Please keep the legacy scheduler enabled
+   until the issue is fully resolved.
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
+* Fix a regression in the new scheduler temporarily by setting the default for the configuration property ``use-legacy-scheduler`` to ``true``.
+* Improve coordinator reliability by adding support to cap the number of total tasks running in a coordinator by pausing scheduling.
+  This can be enabled by the configuration property ``experimental.max-total-running-task-count-to-not-execute-new-query``.
+* Improve the scale writer heuristics by considering overall producer buffer utilization. This can be enabled the session property
+  ``optimized_scale_writer_producer_buffer`` and the configuration property ``optimized-scale-writer-producer-buffer``.
+* Improve end point ``v1/resourceGroupState`` to supporting filtering of resource groups that are dynamically generated.
+  It is possible to filter the dynamically generated resource groups.
+* Improve connection pooling to avoid running out of sockets.
+* Add :ref:`khyperloglog_type` type and related functions.
+* Add support forcing streaming exchange for Mark Distinct. This can be enabled by the session property ``use_stream_exchange_for_mark_distinct``
+  and the configuration property ``query.use-streaming-exchange-for-mark-distinct``. (:pr:`14216`).
+
+
+Geospatial Changes
+__________________
+* Improve storage efficiency for Bing tiles by storing Bing tiles as ``BIGINT``. This also reduces bucket skew in certain cases.
+* Add support for spatial joins for join condition ``ST_Distance(p1, p2) < r``.
+
+Hive Changes
+____________
+* Add ``ZSTD`` support for writing ``ORC`` and ``DWRF`` files. This can be enabled by setting session property ``hive.compression_codec`` to ``ZSTD``.
+
+Verifier Changes
+________________
+* Add support for verifying ``SELECT`` queries that produce ``DATE`` or ``UNKNOWN`` (null) columns, or structured typed columns with ``DATE`` or ``UNKNOWN``.
+* Add support for auto-resolving control check query failures due to ``EXCEEDED_TIME_LIMIT``.
+* Add determinism analysis support for simple queries with top-level ``ORDER BY LIMIT`` clause. (:pr:`14181`).
+
+SPI Changes
+___________
+* Add parameter ``AccessControlContext context`` to all methods in ``SystemAccessControl``.
```

Comment:
I don't think that we need to pass the name of the variable `context`? Just mentioning the type should be enough.